fabric-rendering-v1/src/client/java/net/fabricmc/fabric/mixin/client/rendering/ClearableRenderStateMixin.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.client.rendering;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import net.minecraft.client.render.item.ItemRenderState;
+import net.minecraft.client.render.state.SkyRenderState;
+import net.minecraft.client.render.state.WeatherRenderState;
+import net.minecraft.client.render.state.WorldBorderRenderState;
+import net.minecraft.client.render.state.WorldRenderState;
+
+import net.fabricmc.fabric.api.client.rendering.v1.FabricRenderState;
+
+@Mixin({
+		ItemRenderState.class,
+		ItemRenderState.LayerRenderState.class,
+		WorldRenderState.class,
+		WeatherRenderState.class,
+		WorldBorderRenderState.class,
+		SkyRenderState.class
+})
+public class ClearableRenderStateMixin {
+	@Inject(method = "clear", at = @At("TAIL"))
```

Comment:
This mixin doesnt seem to be working in production, as its targgeting many diffrent methods, that each have a different intermediary name. Ill fix this now by splitting it into many mixins.